tendermint/src/block/header.rs

```diff
@@ -74,7 +74,7 @@ pub struct Header {
 /// application.
 ///
 /// <https://github.com/tendermint/tendermint/blob/master/docs/spec/blockchain/blockchain.md#version>
-#[derive(Serialize, Deserialize, Clone, Debug)]
+#[derive(Serialize, Deserialize, Clone, Debug, Default)]
```

Comment:
I think `Default` supposed to some universal default value for the data type, like unit in a monoid, should not change any time in the future (it would be a breaking change).

Reply:
Removed.